lib/ansible/config/routing.yml

```
@@ -2989,59 +2991,59 @@ plugin_routing:
     sf_volume_manager:
       redirect: community.general.sf_volume_manager
     netapp_e_alerts:
-      redirect: community.general.netapp_e_alerts
+      redirect: netapp_eseries.santricity.netapp_e__alerts
     netapp_e_amg:
-      redirect: community.general.netapp_e_amg
+      redirect: netapp_eseries.santricity.netapp_e__amg
     netapp_e_amg_role:
-      redirect: community.general.netapp_e_amg_role
+      redirect: netapp_eseries.santricity.netapp_e__amg_role
```

```suggestion
      redirect: netapp_eseries.santricity.netapp_e_amg_role
```
https://github.com/netappeseries/santricity/tree/master/plugins/modules
it looks like there should be one underscore

---

@gundalow i meant all netapp_eseries.santricity.* modules have the same problem:)